components/Tile.jsx

```diff
@@ -1,5 +1,10 @@
-const Tile = ({ children, classes }) => {
-  return <span className={`bg-white rounded-lg p-5 drop-shadow-md ${classes}`}>{children}</span>;
+const Tile = (props) => {
+  return <span className={`bg-white ${props.edgeRounding} ${props.padding} drop-shadow-md ${props.classes}`}>{props.children}</span>;
 };
 
+Tile.defaultProps = {
+  edgeRounding: "rounded-lg",
+  padding: "p-4",
+}
+
```

Comment:
Same thing as my other comment - could we just keep the default values straight in className, and just overwrite them in `props.classes`? This would reduce the amount of props we need to keep track of